sharedse/src/commonJvmMain/kotlin/com/ustadmobile/port/sharedse/contentformats/ContentTypeUtil.kt

```diff
@@ -30,7 +30,7 @@ val mimeTypeSupported: List<String> = CONTENT_PLUGINS.flatMap { it.mimeTypes.asL
 val extSupported: List<String> = CONTENT_PLUGINS.flatMap { it.fileExtensions.asList() }
 
 suspend fun extractContentEntryMetadataFromFile(file: File, db: UmAppDatabase, plugins: List<ContentTypeFilePlugin> = CONTENT_PLUGINS): ImportedContentEntryMetaData? {
-    plugins.forEach {
+   /* plugins.forEach {
```

Comment:
ContentTypeUtil should also be deleted...

Reply:
done